polaris/ocean/tests/single_column/ideal_age/__init__.py

```diff
@@ -0,0 +1,53 @@
+import os
+
+from polaris import TestCase
+from polaris.ocean.tests.single_column.forward import Forward
+from polaris.ocean.tests.single_column.initial_state import InitialState
+from polaris.ocean.tests.single_column.viz import Viz
+from polaris.validate import compare_variables
+
+
+class IdealAge(TestCase):
+    """
+    The default test case for the single column test group simply creates
+    the mesh and initial condition, then performs a short forward run on 4
+    cores.
+    """
+    def __init__(self, test_group, resolution, ideal_age=True):
+        """
+        Create the test case
+        Parameters
+        ----------
+        test_group : polaris.ocean.tests.single_column.SingleColumn
+            The test group that this test case belongs to
```

Comment:
`resolution` needs to be added to the docstring and as an Attribute to the class's docstring.

Reply:
```suggestion
        test_group : polaris.ocean.tests.single_column.SingleColumn
            The test group that this test case belongs to

        resolution : float
            The horizontal resolution in km
```

Reply:
added